readme.md

```diff
@@ -1,18 +1,49 @@
 # Spring PetClinic Sample application [![setup automated](https://img.shields.io/badge/Gitpod-ready_to_code-orange?logo=gitpod)](https://gitpod.io/from-referrer/)
 
-Start coding in a [ready-to-code development environment](https://www.gitpod.io):
+[Petclinic](https://github.com/spring-projects/spring-petclinic) is a [Spring Boot](https://spring.io/guides/gs/spring-boot/) application built using [Maven](https://spring.io/guides/gs/maven/).
 
-<a href="https://gitpod.io/from-referrer/" style="padding: 10px;">
-    <img src="https://gitpod.io/button/open-in-gitpod.svg" width="150" alt="Push" align="center">
-</a>
+## Table of Contents
+
+ - [Understanding the Spring Petclinic application with a few diagrams](#understanding-the-spring-petclinic-application-with-a-few-diagrams)
+ - [Run petclinic on Gitpod](#run-petclinic-on-gitpod)
+ - [Get Started With Your Own Project](#get-started-with-your-own-project)
+ - [Community support](#community-support)
 
 ## Understanding the Spring Petclinic application with a few diagrams
 <a href="https://speakerdeck.com/michaelisvy/spring-petclinic-sample-application">See the presentation here</a>
 
 <img width="1042" alt="petclinic-screenshot" src="https://cloud.githubusercontent.com/assets/838318/19727082/2aee6d6c-9b8e-11e6-81fe-e889a5ddfded.png">
 
+## Run petclinic on Gitpod
+
+Start coding in a [ready-to-code development environment](https://www.gitpod.io)
+
+Setup Gitpod integration with Jetbrains Gateway using the instructions given in this [doc](https://www.gitpod.io/docs/ides-and-editors/intellij).
+
+Click the button below to start a new workspace on Gitpod which will automatically open the workspace in IntelliJ IDEA through JetBrains Gateway.  
+
+
+<a href="https://gitpod.io/from-referrer/" style="padding: 10px;">
+    <img src="https://gitpod.io/button/open-in-gitpod.svg" width="150" alt="Push" align="center">
+</a>
+
+
 Or you can run it from Maven directly using the Spring Boot Maven plugin. If you do this it will pick up changes that you make in the project immediately (changes to Java source files require a compile as well - most people use an IDE for this):
 
 ```
 ./mvnw spring-boot:run
 ```
+
+## Get Started With Your Own Project
+
+To get started with Spring Boot on Gitpod, add a [`.gitpod.yml`](./.gitpod.yml) file which contains the configuration to improve the developer experience on Gitpod. To learn more, please see the [Getting Started](https://www.gitpod.io/docs/getting-started) documentation.
```

Comment:
**Suggestion:** Swap to the new java doc? It'd be nice if there were spring specific info in there too, but we can add soon!

**Thought:** There are lots of configurations in Gitpod outside of the `.gitpod.yml`, I like the idea of giving a concrete next action, but wonder if we're better off just linking to the docs here. 

```suggestion
To get started with Spring Boot and Java on Gitpod, see the [Java Language Guide](https://www.gitpod.io/docs/languages/java).
```